### DISCUSSION TITLE
运营后台的交互设计【分享】 - Tw93

### DISCUSSION THREAD

**Discussion by tw93** [tw93/tw93.github.io#100] [General]
https://tw93.github.io/2020-03-18/how-to-do-design.html 

 这里是Tw93的技术博客，用于我的学习总结和研究，主要方向是前端，热爱分享、开源，最近在使用react和weex。Share给阿里同学分享的关于中后台交互设计的思考

**Comment by Yunkou** [comment]
运营后台很少诱人关注体验，都是能用就行，很遗憾没听到现场演讲

**Reply by tw93** [reply]
惭愧 当时思考其实比较初级的
